Aztec/Classes/TextKit/TextView.swift

```diff
@@ -202,9 +208,13 @@ open class TextView: UITextView {
 
     // MARK: - Init & deinit
 
-    public init(defaultFont: UIFont, defaultMissingImage: UIImage) {
+    public init(
+        defaultFont: UIFont,
+        defaultParagraphStyle: ParagraphStyle = ParagraphStyle.default,
+        defaultMissingImage: UIImage) {
```

Comment:
Since this initializer is multiline, maybe move the `{` to the next one?

Reply:
AFAIK `{` go on the same line as the declaration?

Reply:
I've gotta check on the coding standard. When the `{ Initializer, Method }`'s signature is multiline, i always promote the `{` to a new line.

Reason is: the method parameter ends up having the exact same indentation as the method's body itself, and promoting the `{` improves readability.

```
 public init(
         someParameter1: Any,
         someParameter2: Any,
         someParameter3: Any) {        
         self.defaultFont = defaultFont
```